docker-compose.yml

```diff
@@ -112,6 +112,34 @@ services:
       - ipfs
     command: /wait-for-it.sh -h rabbit -p 5672 -t 30 -- /wait-for-it.sh -h mongo -p 27017 -- npm start
 
+  poet-node_blockchain-writer:
+    build:
+      context: .
+      dockerfile: ./Docker/Dockerfile.blockchainWriter
+    depends_on:
+      - mongo
+      - rabbit
+      - bitcoind-1
+      - bitcoind-2
+    environment:
+      - POET_SERVICE=blockchain-writer
+      - BITCOIN_URL=bitcoind-1
+      - BITCOIN_URL_B=bitcoind-2
+      - RABBITMQ_URL=amqp://admin:adminPass@rabbit
+      - MONGODB_HOST=mongo
+      - MONGODB_USER=poet
+      - MONGODB_PASSWORD=poetPass
+      - GH_TOKEN=${GH_TOKEN:-''}
+      - NPM_TOKEN=${NPM_TOKEN:-''}
+    volumes:
+      - ./scripts/wait-for-it.sh:/wait-for-it.sh
+    links:
+      - mongo
+      - rabbit
+      - bitcoind-1
+      - bitcoind-2
+    command: /wait-for-it.sh -h rabbit -p 5672 -t 30 -- /wait-for-it.sh -h mongo -p 27017 -- npm run start:blockchain-writer
```

Comment:
@krobi64 @WesleyCharlesBlake if we're overriding the command here, is the new dockerfile needed at all?

Reply:
Nope. We can override the commands in the deployment manifests. The same approach I used for the regtest miner. So we're good here.

Reply:
So I don't need a separate Dockerfile? Nice.

Reply:
Correct. Provided I have the commands I can override the Docker file entry point